packages/traceability/issuer/contracts/RegistryExtended.sol

```diff
@@ -120,7 +120,9 @@ contract RegistryExtended is Registry {
 			Certificate memory cert = certificateStorage[_ids[i]];
 			topics[i] = cert.topic;
 
-            safeTransferFrom(_from[i], _to[i], _ids[i], _values[i], _data[i]);
+			if (_from[i] != _to[i]) {
+            	_safeTransferFrom(_from[i], _to[i], _ids[i], _values[i], _data[i]);
+			}
 
 			_burn(_to[i], _ids[i], _values[i]);
```

Comment:
Seems like we have to move `_burn(_to[i], _ids[i], _values[i]);` into `if` statement

Reply:
And use spaces instead of tab please 😄

Reply:
The `burn` method should stay outside the `if` clause, because we always burn the tokens - regardless whether the `from` and `to` addresses are the same.

Reply:
We just skip transferring the tokens if the `to` and `from` are the same address, because a transfer is unnecessary.

Reply:
Ok, seems reasonable